App/SB/views/OnBoarding/index.js

```diff
@@ -26,29 +27,33 @@ export default class OnBoarding extends Component {
 
     return (
       <Fragment>
-        <View style={styles.onBoardingContainer}>
-          {pages.map(page => (
-            <Fragment key={page.image}>
-              {page.order === step &&
-                <Fragment>
-                  <View style={styles.imageContainer}>
-                    <Image source={page.image} />
-                  </View>
-                  <View style={styles.titleContainer}>
-                    <Text style={styles.title}>{page.title}</Text>
-                  </View>
-                  <Text style={styles.subtitle}>{page.subTitle}</Text>
-                </Fragment>}
-            </Fragment>
-          ))}
-        </View>
-        <Footer
-          currentPageIndex={step}
-          pages={pages}
-          onNext={this.onNext}
-          onSkip={this.onSubmit}
-          onSubmit={this.onSubmit}
-        />
+        <SafeAreaView style={styles.safeContainerTop} />
+        <SafeAreaView style={styles.safeContainerBottom}>
```

Comment:
Little hack here: Use two `SafeAreaView`s to support different colors for the top and bottom parts of the screen.